### PR TITLE
Update about page

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

CleanShot 2022-06-24 at 13 48 01

- [x] Add Custom UI back

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -1573,6 +1573,15 @@
           "system_health": {
             "manage": "Manage",
             "more_info": "more info"
+          },
+          "items": {
+            "change_log": "Change Log",
+            "thanks": "Thanks To",
+            "merch": "Merchandise",
+            "feature": "Feature Requests",
+            "bug": "Bug Report",
```
Comment (spacegaier) [inline src/translations/en.json:1582]:
```suggestion
            "bug": "Bug Reports",
```

Since FRs are also plural above

**Comment by bramkragten** [comment]
Should we add a color diff between the version number and the name of the component?

File: src/panels/config/info/ha-config-info.ts [hunk 2]
```diff
@@ -36,8 +97,8 @@ class HaConfigInfo extends LitElement {
 
   protected render(): TemplateResult {
     const hass = this.hass;
-    const customUiList: Array<{ name: string; url: string; version: string }> =
-      (window as any).CUSTOM_UI_LIST || [];
+    // const customUiList: Array<{ name: string; url: string; version: string }> =
+    //   (window as any).CUSTOM_UI_LIST || [];
```
Comment (bramkragten) [inline src/panels/config/info/ha-config-info.ts:101]: ?

Comment (zsarnett) [reply src/panels/config/info/ha-config-info.ts:101]: Yea its in the description above to add this back. on the to do. Making this a draft

File: src/panels/config/info/ha-config-info.ts [hunk 3]
```diff
@@ -11,12 +23,61 @@ import {
...
+  },
+  {
+    name: "thanks",
+    path: "/developers/credits/",
+    iconPath: mdiHandsPray,
+    iconColor: "#3B808E",
+  },
+  {
+    name: "merch",
+    path: "https://home-assistant-store.creator-spring.com",
```
[the remark this answers is not in the document]
Comment (zsarnett) [reply src/panels/config/info/ha-config-info.ts:51]: Good point. Ill add a redirect

File: src/panels/config/info/ha-config-info.ts (hunk 2 above)
Comment (bramkragten) [reply src/panels/config/info/ha-config-info.ts:101]: Missed it hidden under the screenshot 😄

**Comment by zsarnett** [comment]
Requires: https://github.com/home-assistant/home-assistant.io/pull/23199

File: src/translations/en.json [hunk 4]
```diff
@@ -1601,7 +1601,7 @@
           "frontend": "frontend-ui",
           "built_using": "Built using",
           "icons_by": "Icons by",
-          "frontend_version": "Frontend version: {version} - {type}",
+          "frontend_version": "Frontend {version}",
```
Comment (bramkragten) [inline src/translations/en.json:1604]: This will give issues with current translations

**Review by bramkragten — DISMISSED** [review]

**Review by bramkragten — APPROVED** [review]

**Pull request merged**